### PR TITLE
Add localization for UI strings in Malay

### PR DESCRIPTION
**Pull request by stepankuzmin**
Hey 👋

This PR adds translations for UI strings we use in GL JS into Malay. We translated the UI strings automatically using the Google Translate API, and we need to verify that the translation is correct.

To help us verify the translations, you can open the `Files changed` tab on this page and compare the translations with the original file [`src/ui/default_locale.js`](https://github.com/mapbox/mapbox-gl-js/blob/main/src/ui/default_locale.js).

If the translation looks good, you can approve it using the `Review changes` button in the `Files changed` tab and selecting the `Approve` radio button when submitting the review.

To improve the machine translation, you can click the plus sign next to the line you want to enhance and select the suggestion button on the toolbar.

After making the suggestions, you can mark the review as finished using the `Review changes` button and selecting the `Approve` radio button.

Thanks for helping 🙌

| Context | Screenshot |
| :------ | :--------: |
| **ID**: `AttributionControl.ToggleAttribution`<br/>**English**: `Toggle attribution`<br/>**Context**: Tooltip text on `AttributionControl` hover | Screenshot 2022-04-12 at 12 22 53
| **ID**: `AttributionControl.MapFeedback`<br/>**English**: `Map feedback`<br/>**Context**: Accessibility label for the `Improve this map` link in the expanded `AttributionControl`. That label can be read by assistive technology | Screenshot 2022-04-11 at 21 40 33
| **ID**: `FullscreenControl.Enter`<br/>**English**: `Enter fullscreen`<br/>**Context**: Tooltip text on FullscreenControl hover when not in the fullscreen mode | Screenshot 2022-04-12 at 12 31 49
| **ID**: `FullscreenControl.Exit`<br/>**English**: `Exit fullscreen`<br/>**Context**: Tooltip text on `FullscreenControl` hover when in the fullscreen mode | Screenshot 2022-04-12 at 12 31 58
| **ID**: `GeolocateControl.FindMyLocation`<br/>**English**: `Find my location`<br/>**Context**: Tooltip text on `GeolocateControl` hover | Screenshot 2022-04-12 at 12 34 57
| **ID**: `GeolocateControl.LocationNotAvailable`<br/>**English**: `Location not available`<br/>**Context**: Tooltip text on `GeolocateControl` hover when we can't locate user | Screenshot 2022-04-12 at 12 35 08
| **ID**: `LogoControl.Title`<br/>**English**: `Mapbox logo`<br/>**Context**: Accessibility label for the Mapbox logo image. That label can be read by assistive technology | Screenshot 2022-04-12 at 12 38 52
| **ID**: `Map.Title`<br/>**English**: `Map`<br/>**Context**: Accessibility label for the map canvas element on the webpage. That label can be read by assistive technology | Screenshot 2022-04-12 at 12 42 03
| **ID**: `NavigationControl.ResetBearing`<br/>**English**: `Reset bearing to north`<br/>**Context**: Tooltip text on bearing button in `NavigationControl` | Screenshot 2022-04-12 at 12 45 06
| **ID**: `NavigationControl.ZoomIn`<br/>**English**: `Zoom in`<br/>**Context**: Tooltip text on plus button in `NavigationControl` | Screenshot 2022-04-12 at 12 45 56
| **ID**: `NavigationControl.ZoomOut`<br/>**English**: `Zoom out`<br/>**Context**: Tooltip text on minus button in `NavigationControl` | Screenshot 2022-04-12 at 12 46 02
| **ID**: `ScrollZoomBlocker.CtrlMessage`<br/>**English**: `Use ctrl + scroll to zoom the map`<br/>**Context**: Text on map blocker alert when cooperative gestures are enabled | Screenshot 2022-04-12 at 13 08 03
| **ID**: `ScrollZoomBlocker.CmdMessage`<br/>**English**: `Use ⌘ + scroll to zoom the map`<br/>**Context**: Text on map blocker alert when cooperative gestures are enabled | Screenshot 2022-04-12 at 13 08 03
| **ID**: `TouchPanBlocker.Message`<br/>**English**: `Use two fingers to move the map`<br/>**Context**: Text on map blocker alert when cooperative gestures are enabled | IMG_A42AFAB7325C-1

### CODE REVIEW

File: src/ui/locales/ms.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+// @flow
+
+const locale = {
+    "AttributionControl.ToggleAttribution": "Togol atribusi",
+    "AttributionControl.MapFeedback": "Maklum balas peta",
+    "FullscreenControl.Enter": "Masukkan skrin penuh",
```
Comment (naimhasim) [inline src/ui/locales/ms.js:6]: change to `"Masuk ke skrin penuh"`

File: src/ui/locales/ms.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    "FullscreenControl.Exit": "Keluar dari skrin penuh",
+    "GeolocateControl.FindMyLocation": "Cari lokasi saya",
+    "GeolocateControl.LocationNotAvailable": "Lokasi tidak tersedia",
```
Comment (naimhasim) [inline src/ui/locales/ms.js:9]: change to `"Lokasi tak tersedia"`

File: src/ui/locales/ms.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    "LogoControl.Title": "Logo peti peta",
```
Comment (naimhasim) [inline src/ui/locales/ms.js:10]: `"logo Mapbox"`

File: src/ui/locales/ms.js [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+    "Map.Title": "Peta",
+    "NavigationControl.ResetBearing": "Tetapkan semula galas ke utara",
```
Comment (naimhasim) [inline src/ui/locales/ms.js:12]: change to `"Set semula galas ke utara"`

File: src/ui/locales/ms.js [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+    "NavigationControl.ZoomIn": "Mengezum masuk",
```
Comment (naimhasim) [inline src/ui/locales/ms.js:13]: change to `"Zum masuk"`

File: src/ui/locales/ms.js [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+    "NavigationControl.ZoomOut": "Zum keluar",
+    "ScrollZoomBlocker.CtrlMessage": "Gunakan ctrl + tatal untuk mengezum peta",
```
Comment (naimhasim) [inline src/ui/locales/ms.js:15]: change to `"Gunakan ⌘ + tatal untuk mengezum peta"`

`"⌘"` or `"ctrl"` modifier depends on OS (?)

File: src/ui/locales/ms.js [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+    "ScrollZoomBlocker.CmdMessage": "Gunakan ⌘ + tatal untuk mengezum peta",
```
Comment (naimhasim) [inline src/ui/locales/ms.js:16]: `"⌘"` or `"ctrl"` modifier depends on OS (?)

**Review by naimhasim — APPROVED** [review]
Hi, local from 🇲🇾  here. I'm highly suggesting the feedback changes.

Reference: https://prpm.dbp.gov.my/CarianBestari

**Comment by stepankuzmin** [comment]
Thanks for the help, @naimhasim!

**Pull request merged**